src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1028,9 +1029,22 @@
                     Condition="'%(PackageFile.IsSourceCodeFile)'!='true'" />
     </ItemGroup>
   </Target>
+  
+  <Target Name="GetPackageReport"
```

Comment:
I don't see it chained on ValidatePackage.  In any case I would prefer it always run for pkgproj builds.